ansible/roles/hadoop/tasks/main.yml

```diff
@@ -34,6 +34,12 @@
   with_items:
     - workers
   when: hadoop_major_version == '3'
+
+# This is currently needed to run hadoop with Java 11 (see https://github.com/apache/fluo-muchos/issues/266)
+- name: "Copy javax.activation-api (when Hadoop 3 and Java 11 are used)"
+  synchronize: src={{ user_home }}/mvn_dep/ dest={{ hadoop_home }}/share/hadoop/common/lib/
+  when: hadoop_major_version == '3' and java_product_version == 11
```

Comment:
Would it make sense to do `ava_product_version >= 11`?

Reply:
Sure, will make that change and push!

Reply:
hold off on merging till I do this, I will accordingly have to handle the enumeration of Java 12, 13 and 14 in the config.py to really make this meaningful.

Reply:
ok

Reply:
hi @keith-turner I made a few updates to the overall logic of Java 11 and 1.9. Regarding the above condition in Ansible, I prefer to leave it as-is. It looks like the [fix](https://issues.apache.org/jira/browse/HADOOP-15775) for the java activation API issue is due to be released with Hadoop 3.3, so hopefully this fix (of copying the javax dependency) will not be needed in the longer term. And I'm also assuming that anyone using Java 12 or above will probably need a higher version of Hadoop than 3.3 - I did not see any JIRAs tracking this ask for Java 12.

Reply:
The Java 11 upgrade was a disruptive upgrade.  Hopefully transitioning to the next Java LTS will be smoother.